lib/quickbooks/model/batch_request.rb

```diff
@@ -23,6 +23,7 @@ class BatchItemRequest
     xml_accessor :sales_receipt, from: "SalesReceipt", as: Quickbooks::Model::SalesReceipt
     xml_accessor :time_activity, from: "TimeActivity", as: Quickbooks::Model::TimeActivity
     xml_accessor :vendor, from: "Vendor", as: Quickbooks::Model::Vendor
+    xml_accessor :query, from: "Query"
```

Comment:
I believe here we could use `xml_accessor :query, from: "Query", as: Quickbooks::Model::QueryResponse` and then below we would just need to `bir.operation = operation unless operation == "query"`.